code/modules/food_and_drinks/machinery/microwave.dm

```diff
@@ -91,23 +96,92 @@
 		"[span_notice("- Capacity: <b>[max_n_of_items]</b> items.")]\n"+\
 		span_notice("- Cook time reduced by <b>[(efficiency - 1) * 25]%</b>.")
 
+#define MICROWAVE_INGREDIENT_OVERLAY_SIZE 24
+
+/obj/machinery/microwave/update_overlays()
+	// When this is the nth ingredient, whats its pixel_x?
+	var/static/list/ingredient_shifts = list(
+		-3,
+		0,
+		-6,
+		1,
+		-7,
+		-1,
+		-5,
```

Comment:
Don't forge to SKYRAT EDIT these, this is a TG file we're changing from the original :b

Though I wonder if we can re-define list/ingredient_shifts in a future file or not. IDK how lists work, or if it'd apply before or after the offsets are actually applied...

Reply:
what I get for adding a last second change at 5am, hah